.doc_gen/metadata/serverless_metadata.yaml

```diff
@@ -481,6 +481,14 @@ serverless_DynamoDB_Lambda:
             - description: Consuming a &DDB; event with &LAM; using JavaScript.
               snippet_files:
                 - integration-ddb-to-lambda/example.js
+    Typescript:
```

Comment:
For future reference, please don't add this metadata in. We (the Docs team) will take care of that.